### DISCUSSION TITLE
Cursor Freezing Consistently with "High Precision Mouse" Enabled

### DISCUSSION THREAD

**Discussion by Boo-BooLazer** [ppy/osu#19914] [Q&A]
I posted a discussion post about a month ago pertaining to this same issue. Am still having the exact same problems and I also noticed that another user is having similar issues to me as well. I don't believe that this problem is on my end as I have been using lazer for years and have never had this problem until recently. Here's the link to the original post, https://github.com/ppy/osu/discussions/19655#discussion-4289676 
Here is a link to the post I mentioned above, 
https://github.com/ppy/osu/discussions/19588#discussion-4281558

For your information, I am currently on version 2022.821.0

Here are my current logs, but you can find my original logs in the original post.

[performance.log](https://github.com/ppy/osu/files/9396918/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9396919/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9396920/updater.log)
[database.log](https://github.com/ppy/osu/files/9396921/database.log)
[network.log](https://github.com/ppy/osu/files/9396922/network.log)

EDIT: I forgot to mention that I just reinstalled Wacom's most updated drivers for my pen tablet. Here is the website to download, https://www.wacom.com/en-us/support/product-support/drivers. My tablet is the CTH-460.

**Comment by peppy** [comment]
Please take a video showing the issue occurring while having the full frame graphs visible (ctrl+f11 twice to show these).

**Reply by Boo-BooLazer** [reply]
Hi, I would love to take a video but githubs upload limit is so small, I can't upload a video of reasonable quality.

**Reply by peppy** [reply]
You can use youtube or anything else.

**Comment by Boo-BooLazer** [comment]
https://user-images.githubusercontent.com/94070063/186219601-d7cc1de6-08f3-434e-a86b-3715145e4b0b.mp4

**Reply by Boo-BooLazer** [reply]
I got a good enough video here. Every time the cursor freezes after I enable High Precision Mouse it is not me. I am able to put in inputs such as Mouse1 through the tablet, but I can't put any inputs regarding the X or Y axis on my tablet.

**Reply by peppy** [reply]
Tracking at #19960.
